internal/pkg/term/selector/errors.go

```diff
@@ -13,3 +18,51 @@ var (
 	// ErrSubnetsNotFound is returned when no existing subnets are found.
 	ErrSubnetsNotFound = errors.New("no existing subnets found")
 )
+
+// ErrNoWorkloadInApp occurs when there is no workload in an application.
+type ErrNoWorkloadInApp struct {
```

Comment:
can we add unit tests for these errors 🥺

Reply:
done!